### PR TITLE
Symantec: add Symantec EPP integration

### PR DESCRIPTION
**Pull request by squioc**
add documentation about Symantec EPP integration

### CODE REVIEW

File: docs/xdr/features/collect/integrations/endpoint/symantec_epp.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+In this guide, you will configure your Symantec Endpoint Protection Manager or your Symantec Endpoint Security to forward events through syslog.
+
+### Prerequisites
+
+An internal log concentrator (Rsyslog) is required to collect and forward events to SEKOIA.IO.
+
+### Enable Syslog forwarding
+
+Log on the console of our management server and follow [this guide](https://techdocs.broadcom.com/us/en/symantec-security-software/endpoint-security-and-management/endpoint-protection/all/Monitoring-Reporting-and-Enforcing-Compliance/viewing-logs-v7522439-d37e464/exporting-data-to-a-syslog-server-v8442743-d15e1107.html) to enable logs forwarding and provide the ip, the transport protocol (recommending `TCP`) and the listening port (`514`) of the concentrator.
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/symantec_epp.md:21]:
```suggestion
Log on the console of our management server and follow [this guide](https://techdocs.broadcom.com/us/en/symantec-security-software/endpoint-security-and-management/endpoint-protection/all/Monitoring-Reporting-and-Enforcing-Compliance/viewing-logs-v7522439-d37e464/exporting-data-to-a-syslog-server-v8442743-d15e1107.html) to enable logs forwarding and provide the IP, the transport protocol (we are recommending `TCP`) and the listening port (`514`) of the concentrator.
```

File: mkdocs.yml [hunk 2]
```diff
@@ -434,6 +435,8 @@ plugins:
       integrations/sentinelone_deepvisibility.md: xdr/features/collect/integrations/endpoint/sentinelone_deepvisibility.md
       integrations/sophos_edr.md: xdr/features/collect/integrations/endpoint/sophos_edr.md
       integrations/sophos_fw.md: xdr/features/collect/integrations/network/sophos_fw.md
+      integrations/symantec-endpoint-protection.md: xdr/features/collect/integrations/endpoint/symantec_epp.md
+      integrations/symantec_endpoint_protection.md: xdr/features/collect/integrations/endpoint/symantec_epp.md
```
Comment (PierrickV) [inline mkdocs.yml:439]:
```suggestion
```

Comment (PierrickV) [reply mkdocs.yml:439]: duplicated line

**Review by PierrickV — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (squioc) [reply mkdocs.yml:439]: The first line use dash ('-') when the second one use underscore ('_')

**Pull request merged**